.github/workflows/ci.yaml

```diff
@@ -137,6 +95,14 @@ jobs:
     runs-on: [self-hosted, Linux]
     container: lampepfl/dotty:2020-04-24
 
+    steps:
```

Comment:
community_build task should probably be removed and not just stubbed.

Reply:
I have removed it.